examples/k8sPatches/usecases/initcontainerpatch.yaml

```diff
@@ -0,0 +1,8 @@
+kind: "Deployment"
```

Comment:
usecases directory? better if we can recommend a way for maintaining their service snippets

Reply:
@srujankujmar